optaplanner-core/src/main/java/org/optaplanner/core/impl/score/director/drools/testgen/reproducer/TestGenCorruptedScoreReproducer.java

```diff
@@ -66,10 +66,10 @@ public boolean isReproducible(TestGenKieSessionJournal journal) {
         } catch (RuntimeException e) {
             if (e.getMessage() != null && e.getMessage().startsWith("No fact handle for ")) {
                 // this is common when removing insert of a fact that is later updated - not interesting
-                logger.debug("    Can't remove insert: {}", e.toString());
+                logger.debug("    Can't remove insert: {}", (Object) e);
```

Comment:
@yurloc Thank you for noticing this. Those subset of changes need to be undone. Do you want to do the honors? (happy to do them myself too)

Is there a difference between e.toString() and e.getMessage()? Should we consider using getMessage() over toString()?